core/src/main/java/com/linecorp/armeria/common/MediaTypeSet.java

```diff
@@ -60,16 +60,36 @@
 
     private final MediaType[] mediaTypes;
 
+    /**
+     * Returns the {@link MediaTypeSet} which is composed of the specified {@link MediaType}s.
+     */
+    public static MediaTypeSet of(MediaType... mediaTypes) {
+        return of(ImmutableList.copyOf(requireNonNull(mediaTypes, "mediaTypes")));
+    }
+
+    /**
+     * Returns the {@link MediaTypeSet} which is composed of the specified {@link MediaType}s.
+     */
+    public static MediaTypeSet of(Iterable<MediaType> mediaTypes) {
+        return new MediaTypeSet(ImmutableList.copyOf(requireNonNull(mediaTypes, "mediaTypes")));
```

Comment:
How about checking if `mediaTypes` is already a `MediaTypeSet`?

Reply:
That's good idea!